storey/steps/partition.py

```diff
@@ -0,0 +1,29 @@
+from collections import namedtuple
+from typing import Callable, Any
+
+from storey import Flow, Event
+from storey.dtypes import _termination_obj
+
+Partitioned = namedtuple("Partitioned", ["left", "right"], defaults=[None, None])
+
+
+class Partition(Flow):
+    """
+    Partitions events by calling a predicate function on each event. Each processed event results in a `Partitioned`
+    namedtuple of either (left=Event, right=None) or (left=None, right=Event).
```

Comment:
It's unclear whether True means left or right... I mean it's clear from the code, but not from the signature/docstring.